nautobot/core/celery/backends.py

```diff
@@ -64,3 +69,32 @@ def _get_extended_properties(self, request, traceback):
             )
 
         return extended_props
+
+    def prepare_exception(self, exc, serializer=None):
```

Comment:
Anything we can do as a test case for this functionality? I'm thinking one of our test Jobs that intentionally throws an exception?